bin/kano-network-hook

```diff
@@ -40,21 +46,33 @@ def send_cpu_id():
 
             # Save a trace of the CPUID so we dont send it over again
             # The UTC timestamp is embedded inside the cpuid file as well
-            utc_time=time.asctime(time.gmtime(time.time()))
             with open(cpuid_filename, 'a') as f:
                 os.utime(cpuid_filename, None)
-                f.write('RaspberryPI model: {}\n{}\n'.format(get_rpi_model(), utc_time))
+                f.write('RaspberryPI model: {}\n{}\n'.format(
+                    get_rpi_model(), utc_time))
 
             # Send this file over the network
             cmdline='curl --retry 5 --retry-delay 3 -T "{}" ' \
                 '--user "cpuname:mko821fgx" http://cpuids.kano.me//cpuserialnos/'.format(cpuid_filename)
             _, _,rc = run_cmd(cmdline)
-            assert (rc==0)
-            sent=True
+            assert (rc == 0)
+            sent = True
         except:
             if os.path.isfile(cpuid_filename): os.unlink(cpuid_filename)
             logger.error('Error sending Kit serial number')
- 
+
+    if os.path.isfile(cpuid_filename_boot):
+        pass
+    else:
+        try:
+            with open(cpuid_filename_boot, 'a') as f:
```

Comment:
probably innocuous, but should'nt it be accessed in `w`rite instead of `a`ppend?

Reply:
True.